values/gatekeeper-operator/gatekeeper-operator.gotmpl

```diff
@@ -3,27 +3,38 @@
 {{- $gk := $v.charts | get "gatekeeper-operator" dict }}
 
 image:
-  repository: quay.io/open-policy-agent/gatekeeper
+  repository: openpolicyagent/gatekeeper
   release: v3.3.0-beta.2
```

Comment:
I'm using latest release as this has an updated documentation as well. `3.2.2` version is maybe not very different and for something like Gatekeeper (which is seldom used) we can deploy latest release now.

Reply:
when you say latest release, I don't know what you are referring to. Can you be more precise in your wording? After all I am addressing a more specific "release" here: stable vs beta.

I just think we should only go for beta if that gives us features we MUST have. Otherwise use stable